src/ERC6551/ImpCookieJar6551.sol

```diff
@@ -29,6 +36,23 @@ contract ImpCookieJar6551 is CookieJarCore, Giver6551 {
         emit GiveCookie(cookieMonster, amount, CookieUtils.getCookieUid(POSTER_UID));
     }
 
+    function eatCookies(
```

Comment:
Also declare the token contract in the event for indexing purposes

Reply:
done.